### DISCUSSION TITLE
LILYGO T-Beam S3

### DISCUSSION THREAD

**Discussion by AWCowan** [lora-aprs/LoRa_APRS_iGate#336] [General]
All
Just starting the journey with LORA and wish to test range using APRS.
I have bought 2 boards LILYGO T-Beam S3 Core 22/11/02 V3.0.
Will these work on the firmware.
Thanks
Andrew GM0UDL

**Comment by peterus** [comment]
Currently not. but i have this changes in plan for the next days as i have the same board at home.

**Comment by AWCowan** [comment]
Thanks very much.

The setup here will be smaller board in house on Collinear.

Second bigger board on mobile with 2/70 antenna on Landrover.

 

Andrew
